service/test/unit/test_application.py

```diff
@@ -97,22 +98,23 @@ def _assert(_):
     @patch('leap.common.events.client')
     @patch('pixelated.application.reactor')
     @patch('pixelated.application.services.Services')
-    def test_should_log_user_out_if_invalid_soledad_token(self, services_mock, reactor_mock, events_mock):
+    def test_that_start_user_agent_binds_to_ssl_if_ssl_options(self, services_mock, reactor_mock, _):
+        # FIXME patch something closer, instead of leap.common
```

Comment:
Something went wrong and `test_should_log_user_out_if_invalid_soledad_token` was deleted whereas `test_that_start_user_agent_binds_to_ssl_if_ssl_options` was duplicated.